### PR TITLE
[Paywall Experiments] Add Paywall features cards

### LINKED ISSUE #2629: Create cards for the new paywall design
See: SEQe6JBDCeegcpa9Bfg5Ww-fi-3_756

### PR DESCRIPTION
**Pull request by mebarbosa**
## Description
- Adds the feature cards for paywall `FEATURE CAROUSEL` type
- See designs: SEQe6JBDCeegcpa9Bfg5Ww-fi-3_756
- Slack discussion: p1723644311582689-slack-C07G1MH890E
- After I connect these cards to the new layout I will be able to tests landscape cases and accessibility

In a later PR I will add these cards to the new layout added here: https://github.com/Automattic/pocket-casts-android/pull/2635

Fixes #2629

## Testing Instructions
- See the `modules/features/account/src/main/java/au/com/shiftyjelly/pocketcasts/account/onboarding/upgrade/paywallfeatures/PaywallCards.kt` file and open the Jetpack Compose preview

## Screenshots or Screencast 

image

## Checklist
- [ ] If this is a user-facing change, I have added an entry in CHANGELOG.md
- [x] Ensure the linter passes (`./gradlew spotlessApply` to automatically apply formatting/linting)
- [ ] I have considered whether it makes sense to add tests for my changes
- [x] All strings that need to be localized are in `modules/services/localization/src/main/res/values/strings.xml`
- [x] Any jetpack compose components I added or changed are covered by compose previews
- [ ] I have updated (or requested that someone edit) [the spreadsheet](https://docs.google.com/spreadsheets/d/107jqrutZhU0fVZJ19SBqxxVKbV2NWSdQC9MFYdLiAxc/edit?usp=sharing) to reflect any new or changed analytics.
 
#### I have tested any UI changes...

- [ ] with different themes
- [ ] with a landscape orientation
- [ ] with the device set to have a large display and font size
- [ ] for accessibility with TalkBack

### CODE REVIEW

**Comment by dangermattic** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">Please add tests for class <code>CardData</code> (or add <code>unit-tests-exemption</code> label to ignore this).</td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          1 Message
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">This PR is still a Draft: some checks will be skipped.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr-check">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: modules/features/account/src/main/java/au/com/shiftyjelly/pocketcasts/account/onboarding/upgrade/paywallfeatures/FeaturesPaywallCards.kt [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+            modifier = Modifier
+                .fillMaxWidth()
+                .background(colorResource(UR.color.coolgrey_90)),
+        ) {
+            Image(
+                painter = painterResource(cardData.imageResId),
+                contentDescription = cardData.contentDescription,
+                modifier = Modifier
+                    .height(cardData.imageHeight)
+                    .padding(top = 20.dp, start = 2.dp, end = 2.dp)
```
Comment (mebarbosa) [inline modules/features/account/src/main/java/au/com/shiftyjelly/pocketcasts/account/onboarding/upgrade/paywallfeatures/FeaturesPaywallCards.kt:60]: I am checking with our designs which top padding fits the best. When I add the same specs in figma it does not look great

**Comment by mebarbosa** [comment]
> The attached images are 3.5 MB in total. Can you run them through an optimization tool like tinypng.com?
> 
> They are also quite big in dimensions when compared to how they are displayed in the UI. For example bookmarks' card is `224 dp` high while the image is `896 px` high. Though this might change if there's a landscape alternative.
> 
> We could probably resize them a bit and then run them through an optimizer. It might be also worth providing multiple resources for different densities.

@MiSikora I added the compressed images that Dom provided to me

**Comment by geekygecko** [comment]
It would be good to follow the existing image folders and include `hdpi` and `xxxhdpi`. I will try to export these tomorrow. I don't believe the images need to be in the `/drawable` folder and the `/drawable-` folders, do they? 

I wondered if we should also try webp images to reduce the size. Can we work together to figure out a good plan for exporting images for the project? Let's have a chat on Slack. 

Screenshot 2024-08-21 at 12 52 52 AM

**Comment by geekygecko** [comment]
I have raised a pull request with some more images.
https://github.com/Automattic/pocket-casts-android/pull/2679/files

**Review by geekygecko — APPROVED** [review]

**Pull request merged**